paasta_tools/vitesscluster_tools.py

```diff
@@ -129,13 +130,58 @@ class ResourceConfigDict(TypedDict, total=False):
     limits: Dict[str, RequestsDict]
 
 
+class HostPathVolume(TypedDict, total=False):
+    path: str
+    type: str
+
+
+class KubernetesVolume(TypedDict, total=False):
+    name: str
+    hostPath: HostPathVolume
```

Comment:
i'm a little surprised that we don't have typed dicts for this type of k8s volume shenanigans - but i guess that's maybe because we never store the volume info as a dict outside of representing the config as seen in soaconfigs

Reply:
Yeah, looks like DockerVolume format is used in the parent classes, and I used this as required for Vitess manifest